doc/flame/components.md

```diff
@@ -37,14 +37,48 @@ If the parent is not mounted yet, then this method will wait in a queue (this wi
 on the rest of the game engine). 
 
 
-## Component
-Usually if you are going to make your own component you want to extend `PositionComponent`, but if
-you want to be able to handle effects and child components but handle the positioning differently
-you can extend the `Component` directly.
+### Priority
```

Comment:
Maybe add a warning here that this describes the current state of affairs, but we're planning to extend the priority mechanism in the future?

Reply:
I don't think we need to mention it before we have discussed how the next priority system should look like.